src/app/personside/infotabs/meldinger/traadvisning/Enkeltmelding.tsx

```diff
@@ -101,6 +102,18 @@ function MeldingLestEtikett({ melding }: { melding: Melding }) {
     );
 }
 
+function SkrevetAv({ melding, rule }: { melding: Melding; rule: Rule }) {
+    if (erMeldingFraBruker(melding.meldingstype)) {
+        return null;
+    }
+    return (
+        <SkrevetAvStyle>
+            <SkrevetAvTekst>Skrevet av:</SkrevetAvTekst>
+            <Tekstomrade rules={[rule]}>{saksbehandlerTekst(melding.skrevetAv)}</Tekstomrade>
```

Comment:
Kan ikke highlightRule bare skrives inn direkte her? Skal vel aldri variere eller?

Reply:
Highlightrule brukes tre steder så tenkte det var greit å ikke lage en egen for _skrevet av_ men at vi har samme i alle `Tekstomrade`

Reply:
Aaha, trodde det var den statiske highlightRule, men det gir jo egentlig ikke mening å bruke 🤦‍♂ 
Men da er det greit. 👍